content/en/contribute/code/core/automated-tests.md

```diff
@@ -161,12 +165,26 @@ Read the failure carefully - it often has really good info but sometimes it's ju
 
 #### Other logs and screenshots
 
-There are logs and screenshots stored in the allure reports. [Here](https://github.com/medic/cht-core/blob/master/TESTING.md#view-the-ci-report) are the instructions to access that information.
+GitHub actions will artifact all files in tests/logs. This is the directory any logs, results, images, etc... should save to if you want to review them if a build fails.
+
+##### View the CI report
+There are logs and screenshots stored in the allure reports when a job failed on the CI. To access to those logs follow these steps:
+- Download the CI run artifact zip file located in the failed build's `Archive Results` section.
+  {{< figure src="archiveResultsSection.png" link="archiveResultsSection.png" class=" center col-4 col-lg-4" >}}
```

Comment:
@esthermmoturi I am not sure if did this correctly, could you please chek?

Reply:
Yes, the image wasn't renderring because the folder with the images is `automated-test` instead of `automated-tests`. I'll make the change.